### PR TITLE
Fixed incorrectly formatted code blocks

### LINKED ISSUE #96: Badly formatted code blocks
Seems like we missed some code blocks: https://gbadev.net/tonc/first.html#ints-versus-non-ints

### PR DESCRIPTION
**Pull request by djedditt**
Fixes #96 and #76

### CODE REVIEW

File: content/first.md [hunk 1]
```diff
@@ -368,39 +368,31 @@ Here are a few examples of code that, while functional, could be improved in ter
 
 Above, I noted that use of non-ints can be problematic. Because this bad habit is particularly common under GBA and NDS code (both homebrew *and* commercial), I'd like to show you an example of this.
```
Comment (AntonioND) [inline content/first.md:370]: I believe this code is special, with some words marked in red so that the reader can see what the text is talking about.

Comment (AntonioND) [reply content/first.md:370]: https://gbadev.net/tonc/first.html#ssec-bad-example

Comment (djedditt) [reply content/first.md:370]: I restored the red highlights, while still introducing code block visual and functionality in [7680e48](https://github.com/gbadev-org/tonc/pull/98/commits/7680e48296225f3bc140f301633fcd8f77cdf940).

**Review by avivace — APPROVED** [review]

**Comment by exelotl** [comment]
There were several more cases where the red highlights got lost during the conversion to Markdown, so I've fixed and pushed those to this branch. Also, a nice thing is that adding the `hljs` class allows syntax highlighting to still work for these code blocks. :D

**Review by avivace — APPROVED** [review]

**Pull request merged**